recipes/cpprestsdk/all/conanfile.py

```diff
@@ -0,0 +1,184 @@
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires.add("openssl/1.1.1e")
+        if not self.options.exclude_compression:
+            self.requires.add("zlib/1.2.11")
+        if not self.options.exclude_websockets:
+            self.requires.add("websocketpp/0.8.1")
+        self.requires.add("boost/1.72.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        if self.settings.os == "iOS":
+            with open('toolchain.cmake', 'w') as toolchain_cmake:
+                if self.settings.arch == "armv8":
+                    arch = "arm64"
+                    sdk = "iphoneos"
+                elif self.settings.arch == "x86_64":
+                    arch = "x86_64"
+                    sdk = "iphonesimulator"
+                sysroot = tools.XCRun(self.settings).sdk_path
+                toolchain_cmake.write('set(CMAKE_C_COMPILER /usr/bin/clang CACHE STRING "" FORCE)\n')
+                toolchain_cmake.write('set(CMAKE_CXX_COMPILER /usr/bin/clang++ CACHE STRING "" FORCE)\n')
+                toolchain_cmake.write('set(CMAKE_C_COMPILER_WORKS YES)\n')
+                toolchain_cmake.write('set(CMAKE_CXX_COMPILER_WORKS YES)\n')
+                toolchain_cmake.write('set(CMAKE_XCODE_EFFECTIVE_PLATFORMS "-%s" CACHE STRING "" FORCE)\n' % sdk)
+                toolchain_cmake.write('set(CMAKE_OSX_ARCHITECTURES "%s" CACHE STRING "" FORCE)\n' % arch)
+                toolchain_cmake.write('set(CMAKE_OSX_SYSROOT "%s" CACHE STRING "" FORCE)\n' % sysroot)
+            os.environ['CONAN_CMAKE_TOOLCHAIN_FILE'] = os.path.join(os.getcwd(), 'toolchain.cmake')
+
+        self._cmake = CMake(self, set_cmake_flags=True)
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_SAMPLES"] = False
+        self._cmake.definitions["WERROR"] = False
+        self._cmake.definitions["CPPREST_EXCLUDE_WEBSOCKETS"] = self.options.exclude_websockets
+        self._cmake.definitions["CPPREST_EXCLUDE_COMPRESSION"] = self.options.exclude_compression
+        self._cmake.definitions["CPPREST_VERSION"] = self.version
+        self._cmake.definitions["OPENSSL_ROOT_DIR"] = self.deps_cpp_info['openssl'].rootpath
+        self._cmake.definitions["OPENSSL_USE_STATIC_LIBS"] = not self.options['openssl'].shared
+        self._cmake.definitions["BOOST_ROOT"] = self.deps_cpp_info["boost"].rootpath
+        self._cmake.definitions["BOOST_INCLUDEDIR"] = self.deps_cpp_info["boost"].include_paths[0]
+        self._cmake.definitions["BOOST_LIBRARYDIR"] = self.deps_cpp_info["boost"].lib_paths[0]
+        self._cmake.definitions["Boost_NO_SYSTEM_PATHS"] = True
+        self._cmake.definitions["Boost_ADDITIONAL_VERSIONS"] = "1.69.0"
```

Comment:
I'm unsure what  this flag does but I assume we need to write here the Boot versions we depend on above which would be 1.72.0

Reply:
It's not even required https://cmake.org/cmake/help/latest/module/FindBoost.html